tools/coverage/README.md

```diff
@@ -14,6 +14,16 @@ The code coverage tool has two major features:
 
 See the [design document](design.md).
 
+## Skipped Files
+
+This tool allows users to specify which files to skip, from reporting and aggregation.
+User can add the git attribute `coverage-excluded=true` to any directory or file to exclude files.
+[Here](https://github.com/knative/serving/blob/master/.gitattributes) is an example.
+
+### Other Automatically Excluded Files
+- Source file (say `abc.go`) without corresponding `_test.go` file (abc_test.go)
+- Any file that has the gitattribute of `linguist-generated=true`
```

Comment:
Does these mean vendor files? Would be nice to say explicitly

Reply:
No. linguist-generated is an attribute used in other knative repos, which is not created or managed by me. I don't think I am the best person to give out an definition for that. I know some of the files with the attr are machine generated and the dev team request us to exempt linguist-generated files from coverage.